### ISSUE TITLE
Online demo

### ISSUE THREAD

**Issue by mxstbr** [carteb/carte-blanche#341]
We should really have a website with an online demo…

**Comment by nikgraf** [comment]
I think the only thing we need to do is disallowing to write variations & only store them locally. Can be a hidden option.

Once that is setup we can easily deploy it to Heroku or AWS and run the development server there …

**Comment by mxstbr** [comment]
We should make that an option anyway, since some people might want to host their CarteBlanche online for their developers to copy source code from (/cc @RStankov), so that pretty much has to be an option.

Sounds good!

**Comment by tleunen** [comment]
I'd like this so bad :)
Also having a way to let people test variations without saving them anywhere, so they can see how things are using their data

**Comment by mxstbr** [comment]
> Also having a way to let people test variations without saving them anywhere, so they can see how things are using their data

Absolutely. Would you be interested to contribute? Would love pair-code this!

**Comment by tleunen** [comment]
I'm interested, but unfortunately I don't have time right now to work on this :/
And I don't know carte-blanche at all, so I'd need to take some time learning how it works before starting to think about the best way to implement the feature.
